### PR TITLE
LPS-75421

### PR DESCRIPTION
**Pull request by ChrisKian**
CC @jonathanmccann

https://issues.liferay.com/browse/LPS-75421

Forwarding to you since user management seems to best fit this module.  Please let me know if you have any questions, or if I should forward to someone else.  Thanks!

### CODE REVIEW

**Comment by drewbrokke** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/foundation/contacts/contacts-service/src/main/java/com/liferay/contacts/service/impl/EntryServiceImpl.java [hunk 1]
```diff
@@ -34,18 +38,55 @@
 	@Override
 	public JSONArray searchUsersAndContacts(
 			long companyId, String keywords, int start, int end)
-		throws PortalException {
+		throws Exception {
```
Comment (drewbrokke) [inline modules/apps/foundation/contacts/contacts-service/src/main/java/com/liferay/contacts/service/impl/EntryServiceImpl.java]: Hey @ChrisKian, which change necessitated that this be an Exception?  I think we can keep it a PortalException, but I want to make sure I'm not missing anything.

Comment (drewbrokke) [reply modules/apps/foundation/contacts/contacts-service/src/main/java/com/liferay/contacts/service/impl/EntryServiceImpl.java]: Ah, I see - it's the `PermissionCheckerFactory.create(user)` call that Jonathan changed later.  That's why my IDE didn't complain about me reverting it back to a PortalException.

**Comment by drewbrokke** [comment]
Pull request sent to https://github.com/brianchandotcom/liferay-portal/pull/52478. Thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by drewbrokke** [comment]
@ChrisKian @jonathanmccann please see my changes on my pull to Brian.

File: modules/apps/foundation/contacts/contacts-service/src/main/java/com/liferay/contacts/service/impl/EntryServiceImpl.java (hunk 1 above)
Comment (ChrisKian) [reply modules/apps/foundation/contacts/contacts-service/src/main/java/com/liferay/contacts/service/impl/EntryServiceImpl.java]: Bingo.  Sorry for the confusion!

Comment (drewbrokke) [reply modules/apps/foundation/contacts/contacts-service/src/main/java/com/liferay/contacts/service/impl/EntryServiceImpl.java]: np :)

**Pull request closed without merging**